prompt.js

```diff
@@ -17,7 +17,9 @@ module.exports = {
             type: "select",
             pageSize: 2,
             choices: [
-                { message: "SASS/SCSS", name: "sass" }
+                { message: "NONE", name: "none"},
+                { message: "SASS/SCSS", name: "sass" },
+                { message: "LESS", name: "less"}
```

Comment:
```suggestion
                { message: "None", name: "none"},
                { message: "Sass/Scss", name: "sass" },
                { message: "Less", name: "less"}
```

diğer prompt'lar ile uyumlu olması için bu şekilde yapalım